firefox/blocked.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE HTML>
+<html lang="en-US" dir="ltr">
+    <head>
+        <meta charset="utf-8">
+
+        <title>Blocked!</title>
+        <link href="/css/min.css" rel="stylesheet">
+        <link rel="stylesheet" href="/css/its-a-trap.css" media="screen" />
+        <script>
+         (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+             (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+                                  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+         })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+         ga('create', 'UA-35433268-54', 'itsatrap.org');
+         ga('send', 'pageview');
+
+        </script>
+    </head>
+
+    <body id="blocked" class="portal-page locale-en-US portal-page">
+
+        <div id="wrapper">
+
+            <div id="doc">
+
+                <div id="nav-access">
+                    <a href="#nav-main">skip to Navigation</a>
+                    <a href="#switch">switch language</a>
+                </div>
+
+                <!-- start #header -->
+                <div id="header">
+                    <div>
+		        <h1><a href="http://www.mozilla.org/firefox/" title="Back to home page">Mozilla Firefox</a></h1>
+		        <a href="http://www.mozilla.org/" class="mozilla">visit <span>mozilla</span></a>
+
+                    </div>
+                </div>
+                <!-- end #header -->
+
+
+                <div id="main-feature">
+	            <h2>Blocked!</h2>
+	            <p>I'm naughty Web content that Firefox doesn't want you to see!</p>
+                </div>
+
+                <div id="main-content">
+
+                    <p>OK, not really, but Firefox can be configured to block certain content. If that feature is enabled in your browser and you are using Firefox 48 or later, you should have seen an error message instead of this page.</p>
```

Comment:
Do we get an error message when we browse to this URL or do we get a blank page instead? (I don't think my Nightly has your patch yet.)